src/main/java/TestFile.java

```diff
@@ -7,8 +7,9 @@ private static void incorrectAssignmentInIfCondition() {
         boolean value = false;
         if (value = true) {
             //do Something
+            value = false;
```

Comment:
[Checkstyle] WARNING: 'if' child have incorrect indentation level 12, expected level should be 6.